tests/test_interpolation.py

```diff
@@ -318,9 +322,15 @@ def test_env_values_are_typed(monkeypatch: Any, value: Any, expected: Any) -> No
 
 def test_env_node_interpolation(monkeypatch: Any) -> None:
     # Test that node interpolations are not supported in env variables.
-    monkeypatch.setenv("my_key", "${other_key}")
-    c = OmegaConf.create(dict(my_key="${env:my_key}", other_key=123))
-    with pytest.raises(InterpolationResolutionError):
+    monkeypatch.setenv("MYKEY", "${other_key}")
+    c = OmegaConf.create(dict(my_key="${env:MYKEY}", other_key=123))
```

Comment:
when you touch `dict()` in tests, can you convert to {} style?
```suggestion
    c = OmegaConf.create({"my_key": "${env:MYKEY}", "other_key": 123})
```

Reply:
Done for the whole file in 85478e4